Instructions on multi-stage docker build lead to error

When using the multi-stage docker build as mentioned [here](https://quarkus.io/guides/building-native-image-guide#creating-a-container-with-a-multi-stage-docker-build), we get the following error:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  32.879 s
[INFO] Finished at: 2019-07-04T16:02:54Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-resources-plugin:2.6:resources (default-resources) on project getting-started: Cannot create resource output directory: /usr/src/app/target/classes -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
The command '/bin/sh -c mvn -f /usr/src/app/pom.xml -Pnative clean package' returned a non-zero code: 1
```

--- a/docs/src/main/asciidoc/building-native-image-guide.adoc
+++ b/docs/src/main/asciidoc/building-native-image-guide.adoc
@@ -285,6 +285,9 @@ In this guide we will use the first stage to generate the native executable usin
 FROM quay.io/quarkus/centos-quarkus-maven:{graalvm-version} AS build
 COPY src /usr/src/app/src
 COPY pom.xml /usr/src/app
+USER root
+RUN chown -R quarkus /usr/src/app
+USER quarkus
 RUN mvn -f /usr/src/app/pom.xml -Pnative clean package
 
 ## Stage 2 : create the docker final image

